src/components/Person/Person.jsx

```diff
@@ -1 +1,52 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+function Person({ person }) {
+  const { 
+    name, 
+    age, 
+    sex, 
+    partnerName, 
+    isMarried 
+  } = person;
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          My name is
+          {' '}
+          {name}
+        </h2>
```

Comment:
Use same approach all over the page
```suggestion
        <h2 className="Person__name">
          {`My name is ${name}`}
        </h2>
```